.github/workflows/trigger-tests-on-nightly-packages.yml

```diff
@@ -0,0 +1,11 @@
+name: Trigger tests for nightly APT packages
+
+on:
+  push:
```

Comment:
Shouldn't this be a cron schedule? Am i missing something?

Reply:
It's under testing I guess (I did the same when testing). I've requested a minor change.

Reply:
Sorry about it, I have tried some new changes, and I would like to know if it works or not -> because of this I have changed it temporary to push.